src/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
```

Comment:
remove this

Reply:
removed

Reply:
you need this file, only wanted you to remove the import of index.css

Reply:
Oh I see, we will add this back in. Sorry about the mix up!

Reply:
It should be added back in now